prowler/providers/aws/services/iam/iam_service.py

```diff
@@ -366,7 +366,34 @@ def __list_policies__(self):
                     if not self.audit_resources or (
                         is_resource_filtered(policy["Arn"], self.audit_resources)
                     ):
-                        policies.append(policy)
+                        policies.append(
+                            Policy(
+                                name=policy["PolicyName"],
+                                arn=policy["Arn"],
+                                version_id=policy["DefaultVersionId"],
+                                type="Custom",
+                                attached=True
+                                if policy["AttachmentCount"] > 0
+                                else False,
+                            )
+                        )
+            list_policies_paginator = self.client.get_paginator("list_policies")
+            for page in list_policies_paginator.paginate(Scope="AWS"):
+                for policy in page["Policies"]:
+                    if not self.audit_resources or (
+                        is_resource_filtered(policy["Arn"], self.audit_resources)
+                    ):
+                        policies.append(
+                            Policy(
+                                name=policy["PolicyName"],
+                                arn=policy["Arn"],
+                                version_id=policy["DefaultVersionId"],
+                                type="AWS",
+                                attached=True
+                                if policy["AttachmentCount"] > 0
+                                else False,
+                            )
+                        )
```

Comment:
Cloud you please move this into a separate function? The only aspect that differs is the `scope` and you can pass it as argument and then use it in the AWS API call and under `policy.type`. Thanks!

Reply:
Done